README.md

```diff
@@ -42,6 +93,18 @@ pb.cache_if! !admin?, "cache-key", expires_in: 10.minutes do
 end
 ```
 
+### merge!
+`merge!` is currently used mainly for `cache!`, but could be used in templates as well. It works with all sorts of data.
```

Comment:
What does merge actually do? What is the output of it when you use it? This seems a bit vague still to me.